docs/data-sources/policy_share.md

```diff
@@ -0,0 +1,30 @@
+---
+subcategory: "Fabric"
```

Comment:
We used "multitenancy" for the corresponding resource: https://github.com/vmware/terraform-provider-nsxt/blob/master/docs/resources/policy_share.md